_data/lang/it/footer-1.yml

```diff
@@ -0,0 +1,50 @@
+- title: Risorse
+  subfolderitems:
+  - name: Riguardo Monero
+    url: resources/about/
+  - name: Moneropedia
+    url: resources/moneropedia/
+  - name: Guide per Sviluppatori
+    url: resources/developer-guides/
+  - name: Guide per l'utente

```

Comment:
Si potrebbe uniformare, tipo 'Guide per Sviluppatori/Guide per Utenti' oppure 'Guide per lo sviluppatore/Guide per l'utente' oppure 'Guide per gli Sviluppatori/Guide per gli Utenti'.

Reply:
It could be standardized, such as 'Guide per Sviluppatori/Guide per Utenti' or 'Guide per lo sviluppatore/Guide per l'utente' or 'Guide per gli Sviluppatori/Guide per gli Utenti'.